feature-libs/cart/quick-order/components/quick-order/quick-order.component.ts

```diff
@@ -97,6 +109,50 @@ export class QuickOrderComponent implements OnInit, OnDestroy {
     this.cartErrors$.next([]);
   }
 
+  clearWarnings(): void {
+    this.cartWarnings$.next([]);
+  }
+
+  clearSuccesses(): void {
+    this.cartSuccesses$.next([]);
+  }
+
+  protected verifyErrors(errors: CartAddEntrySuccessEvent[]): void {
```

Comment:
I would change the name of all `verify*` methods to something like `extractErrors`, `extractWarnings`. That would tell more about what these functions are doing

Reply:
good point